### PR TITLE
Add the sig/pm label to the label configuration

### PR DESCRIPTION
**Pull request by nikhita**
So that `/sig pm` works.

See https://github.com/kubernetes/test-infra/pull/8335#issuecomment-396601037 where it didn't work.

### CODE REVIEW

**Comment by nikhita** [comment]
I'm not sure if adding the label to label configuration is enough for the command to work. :thinking:

**Comment by nikhita** [comment]
cc @kubernetes/sig-contributor-experience-pr-reviews since it adds a new label to `labels.yaml`.

cc @justaugustus @idvoretskyi since it adds the sig/pm label.

File: prow/plugins/label/label_test.go [hunk 1]
```diff
@@ -416,6 +416,26 @@ func TestLabel(t *testing.T) {
 			expectedRemovedLabels: []string{},
 			commenter:             orgMember,
 		},
+		{
+			name:                  "add sig/pm label",
```
Comment (cblecker) [inline prow/plugins/label/label_test.go]: We already have tests for a sig label. You can remove these tests.

File: label_sync/labels.yaml [hunk 2]
```diff
@@ -445,6 +445,12 @@ labels:
     target: both
     prowPlugin: label
     addedBy: anyone
+  - color: d2b48c
+    description: Categorizes an issue or PR as relevant to sig-product-management.
```
Comment (cblecker) [inline label_sync/labels.yaml]: Is it sig-product-management, or just officially sig-pm now? @idvoretskyi

**Comment by idvoretskyi** [comment]
@cblecker sig-pm

**Comment by nikhita** [comment]
@cblecker had forgotten to update the docs generation before :)

**Comment by cblecker** [comment]
/lgtm
💯

**Comment by grodrigues3** [comment]
@krzyzacy @cjwagner can we get an approval here?

**Pull request merged**